python/delta/integration_tests/unity-catalog-commit-coordinator-integration-tests.py

```diff
@@ -54,54 +63,195 @@
+        self.append(MANAGED_CC_TABLE)
+        updated_tbl = read_with_cdf_timestamp(
+            current_timestamp, MANAGED_CC_TABLE).select("id").toDF("id")
+        assertDataFrameEqual(updated_tbl, create_df_with_rows([(4, ), (5, )]))
+
+    # Dataframe Writer V1 Tests #
+    def test_insert_into_append(self):
+        single_col_df = spark.createDataFrame([(4, ), (5, )], schema=["id"])
+        single_col_df.write.mode("append").insertInto(f"{CATALOG_NAME}.{SCHEMA}.{MANAGED_CC_TABLE}")
+        updated_tbl = read(MANAGED_CC_TABLE).toDF("id")
+        assertDataFrameEqual(updated_tbl, create_df_with_rows([(1, ), (2, ), (3, ), (4, ), (5, )]))
+
+    def test_insert_into_overwrite(self):
+        single_col_df = spark.createDataFrame([(5, )], schema=["id"])
+        single_col_df.write.mode("overwrite").insertInto(
+            f"{CATALOG_NAME}.{SCHEMA}.{MANAGED_CC_TABLE}", True)
+        updated_tbl = read(MANAGED_CC_TABLE).toDF("id")
+        assertDataFrameEqual(updated_tbl, create_df_with_rows([(5, )]))
+
+    def test_insert_into_overwrite_replace_where(self):
+        single_col_df = spark.createDataFrame([(5, )], schema=["id"])
+        single_col_df.write.mode("overwrite").option("replaceWhere", "id > 1").insertInto(
+            f"{CATALOG_NAME}.{SCHEMA}.{MANAGED_CC_TABLE}", True)
+        updated_tbl = read(MANAGED_CC_TABLE).toDF("id")
+        assertDataFrameEqual(updated_tbl, create_df_with_rows([(1, ), (5, )]))
+
+    def test_insert_into_overwrite_partition_overwrite(self):
+        single_col_df = spark.createDataFrame([(5,)], schema=["id"])
+        single_col_df.write.mode("overwrite").option(
+            "partitionOverwriteMode", "dynamic").insertInto(
+            f"{CATALOG_NAME}.{SCHEMA}.{MANAGED_CC_TABLE}", True)
+        updated_tbl = read(MANAGED_CC_TABLE).toDF("id")
+        assertDataFrameEqual(updated_tbl, create_df_with_rows([(5,)]))
+
+    def test_save_as_table_append_existing_table(self):
+        single_col_df = spark.createDataFrame(
+            [(4, ), (5, )], schema=StructType([StructField("id", IntegerType(), True)]))
+        single_col_df.write.format("delta").mode("append").saveAsTable(
+            f"{CATALOG_NAME}.{SCHEMA}.{MANAGED_CC_TABLE}")
+        updated_tbl = read(MANAGED_CC_TABLE).toDF("id")
+        assertDataFrameEqual(updated_tbl, create_df_with_rows([(1, ), (2, ), (3, ), (4, ), (5, )]))
+
+
+    # Setting mode to append should work, however cc tables do not allow path based access.
+    def test_save_append_using_path(self):
+        single_col_df = spark.createDataFrame([(4, ), (5, )])
+        # Fetch managed table path and attempt to side-step UC
+        # and directly update table using path based access.
+        tbl_path = spark.sql(
+            f"DESCRIBE formatted {CATALOG_NAME}.{SCHEMA}.{MANAGED_CC_TABLE}").collect()[5].data_type
+        try:
+            single_col_df.write.format("delta").save(mode="overwrite", path=tbl_path)
```

Comment:
```suggestion
            single_col_df.write.format("delta").save(mode="append", path=tbl_path)
```
to match what the test says